config/locales/en_ZA.yml

```diff
@@ -0,0 +1,3094 @@
+en_GB:
```

Comment:
Shouldn't this here be`en_ZA` @luisramos0 ?

Reply:
yes, my bug, I only realized that after fixing it in transifex...
anyway, done.
it's not coming from transifex in this PR:
https://github.com/openfoodfoundation/openfoodnetwork/pull/3945